tests/internal/test_remoteconfig.py

```diff
@@ -147,6 +149,35 @@ def _reload_features(self, metadata, features):
         assert callback.features == {"asm": {"enabled": True}}
 
 
+def test_remote_configuration_check_deprecated_var(caplog):
+    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1")):
+        with warnings.catch_warnings(record=True) as capture:
+
+            # Hack: need to pass an argument to configure so that the processors are recreated
+            get_poll_interval_seconds()
+            assert len(capture) == 0
+
+
+def test_remote_configuration_check_deprecated_var_message(caplog):
+    with override_env(dict(DD_REMOTECONFIG_POLL_SECONDS="0.1")):
+        with warnings.catch_warnings(record=True) as capture:
+
+            # Hack: need to pass an argument to configure so that the processors are recreated
+            get_poll_interval_seconds()
+            assert len(capture) == 1
+            assert str(capture[0].message).startswith("Using environment")
+
+
+def test_remote_configuration_check_deprecated_override(caplog):
+    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1", DD_REMOTECONFIG_POLL_SECONDS="0.1")):
```

Comment:
```suggestion
    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1", DD_REMOTECONFIG_POLL_SECONDS="0.5")):
```

mild suggestion, this will just confirm we pick up the new name value first, and ignore the older value.